review/review.go

```diff
@@ -646,7 +691,7 @@ func (r *Review) AddComment(c comment.Comment) error {
 // review will be added to the 'refs/devtools/archives/reviews' ref prior
 // to being rewritten. That ensures the review history is kept from being
 // garbage collected.
-func (r *Review) Rebase(archivePrevious bool) error {
+func (r *Review) Rebase(archivePrevious, sign bool) error {
```

Comment:
Can you please add a RebaseAndSign method here similarly to how you added for the Repo interface?

That will both prevent backwards-incompatible changes, and also keep the Review API consistent with the Repo API.

The current method body could just be a helper that both `Rebase` and `RebaseAndSign` invoke.

Reply:
On it!